scripts/prepare_electricity_demand.py

```diff
@@ -0,0 +1,117 @@
+r"""
+Inputs
+-------
+url_opsd : str
+    url of raw opsd data of electricty load as .csv
+output_file : str
+    path of output file with prepared data as .csv
```

Comment:
```suggestion
url_opsd : str
    url of raw opsd data of electricty load as .csv
output_file : str
    ``results/_resources/load_profile_time_series.csv``: path of output file with prepared data as .csv
```

Reply:
According to #82, we want to include the filenames in the header, so that they are shown in the docs, which helps the user.